posts/20230223/index.qmd

```diff
@@ -0,0 +1,67 @@
+
+While this abstract seemed reasonably promising to understand the application of ML in real estate evaluation, when I read the paper more closely, I found myself questioning many of the approaches the authors took to answer their central question. Therefore, this blog is structured such that it summarizes the authors’ work as outlined in this paper, and then also offers alternative considerations from the perspective of a reviewer.
+
+## The data  
+The authors wanted to build and compare models for predicting home prices. They posit that previous literature addressed home pricing with hedonic regression models, i.e. linear regression models with specific focus on property outcomes like its individual environmental characteristics, but not necessarily accounting for broader socio-economic factors. With the modeling conducted in this paper, the authors attempted to model home pricing accounting for such factors. However, instead of predicting home pricing as a continous variable as with regression, the authors chose to make home prediction a binary classification problem. The data used in this research consist of ~94,000 rows of publicly available real estate sale data from a single county in Florida, Volusia County.
+
+#### Outcome variable
+The way the authors defined the binary outcome variable for this research was interesting: they took the final sale price of the home and compared it to the appraised government price of the home, assigning the variable 1 if the property’s selling price is above the appraised price (which they call `high price`), and 0 if not (called `low price`). However, the authors were not clear on why this designation was useful to classify. More importantly there was no information on how or when the properties were initially appraised by the government, which makes the comparison to final sale price unclear. Furthermore, the authors don’t provide a tally on how many of the properties are classified as high price, though the authors do show a plot (see paper Fig. 2) that suggests that the total count of high price properties have steadily increased in the 10 years of this data. But despite this fact, appraisal year or sale year are not predictor variables the authors considered. 
+
+Another way the author could have approached the outcome variables was to have compared the initial listing price to the final sale price. In fact, in the abstract the authors tout that the listing price is baseline against which the final sale price would be compared, but the final models differ from this initial assertion. Regardless, this binary outcome variable falls short because the magnitude of difference between any two price comparisons would be lost; a high price differential of $100k would be treated the same as a lower price differential of $5k. 
+
```

Comment:
The sentence starting with "regardless" is the first time you give an opinion. All other comments are factual. Giving your personal opinion as an expert working in this field would make this more interesting!

Reply:
I tried to add more real estate context when explaining the choices in appraisal vs. assessed valuations